standalone/examples_test.go

```diff
@@ -0,0 +1,200 @@
+package standalone
+
+import (
+	"encoding/json"
+	"github.com/fullstorydev/grpcui/testing/testdata"
```

Comment:
ditto about `goimports`

Reply:
I'm confused how this compiles. I don't see any references to `testdata` in here.